src-ui/src/app/components/common/input/date/date.component.ts

```diff
@@ -53,17 +53,11 @@ export class DateComponent
   }
 
   onSuggestionClick(dateString: string) {
-    const parsedNgDate = this.ngbDateParserFormatter.parse(dateString)
-    this.writeValue(this.formatDateAsYYYYMMDD(parsedNgDate))
+    const parsedDate = this.ngbDateParserFormatter.parse(dateString)
+    this.writeValue(`${parsedDate.year}-${parsedDate.month}-${parsedDate.day}`)
```

Comment:
Hey @shamoon,

thanks for your review!

Did you test your change with a date like 1.1.22?
I had the problem, that the date input field expects a date with two digits for month and day, otherwise it did not work for me.

Reply:
Im not seeing that issue but there is a problem with dates being off by 1 day, that is due to using `Date()` which automatically uses timezone (see #957). I pushed another commit to deal with this. Can you test as-is and tell me if youre seeing any issues?

Reply:
I have tested your change. Looks good. I did not see any issues.

Thanks for improving the frontend :)